source/changelog.txt

```diff
@@ -12,6 +12,36 @@
    :depth: 1
    :class: singlecol
 
+v1.2.3
+------
+
+*Released March 10, 2022*
+
+New features:
+
+- :issue:`MONGOSH-1121` - Support the ``commitQuorum`` parameter for the
+  :method:`~db.collection.createIndexes()` method.
+
+Bug fixes:
+
+- Use Rocky Linux 8 instead of CentOS 7 for packaging.
```

Comment:
This doesn’t affect users – if there’s anything to call out here, it’s that .rpm packages now have SHA256 digests, but 99.9 % of users won’t care about that either :)

Reply:
Got it, will remove 👍